internal/osbuild/greenboot_stage.go

```diff
@@ -0,0 +1,14 @@
+package osbuild
+
+type GreenbootConfig struct {
+	MonitorService string `json:"monitor_services,omitempty"`
```

Comment:
the stage in https://github.com/osbuild/osbuild/pull/1086 does define this as an array of strings so that here too